src/helpers/Logger.ts

```diff
@@ -6,30 +6,40 @@ const dateFormat = () => {
   return new Date(Date.now()).toLocaleString()
 }
 
+interface InfoLog {
+  message: string
+  context?: Object
+}
+
 class Logger {
   private logger: winston.Logger;
   private caller: string;
   constructor (caller: string) {
     this.caller = caller
 
     const logger = winston.createLogger({
-      level: 'error',
-      format: winston.format.printf(error => {
-        let message = `[${this.caller}: ${error.source}] | ${dateFormat()} |  ${error.level.toUpperCase()} | ${error.message} | `
-        message = error.thrownError ? `${message} ${JSON.stringify(error.thrownError)} | ` : message
+      level: 'info',
+      format: winston.format.printf(log => {
+        let message = `[${this.caller}: ${log.source}] | ${dateFormat()} |  ${log.level.toUpperCase()} | ${log.message} | `
+        message = log.logDetails ? `${message} ${JSON.stringify(log.logDetails)} | ` : message
         return message
       }),
       transports: [
         new winston.transports.Console(),
-        new winston.transports.File({ filename: `${configs.LOG_FILE_PATH}/error.log` })
+        new winston.transports.File({ filename: `${configs.LOG_FILE_PATH}/error.log`, level: 'error' }),
+        new winston.transports.File({ filename: `${configs.LOG_FILE_PATH}/info.log`, level: 'info' })
       ]
     })
 
     this.logger = logger
   }
 
   logError (source: string, err: Error | HttpError) {
-    this.logger.error(err.message, { source, thrownError: err })
+    this.logger.error(err.message, { source, logDetails: err })
+  }
+
+  logInfo (source: string, info: InfoLog) {
+    this.logger.info(info.message, { source, logDetails: info })
```

Comment:
info logger is added to be used when we need to log an `info` logs, it will be saved to `info.log` and is used in this PR to an info while